### PR TITLE
ABİT SUNGURTEKİN 

### PR DESCRIPTION
**Pull request by abitsungurtekin**
abitsungurtekin

### CODE REVIEW

**Comment by ThePain813** [comment]
Yes this is the owner of the pain131313@gmail.com on issue#326 speaks of (my winnings) I believe someone else has taken my tokens

**Comment by AZIMALAVI** [comment]
عظیم علوی

در تاریخ جمعه ۲۴ فوریهٔ ۲۰۲۳،‏ ۰۷:۱۲ ThePain813 ***@***.***>
نوشت:

**Comment by haticecan37** [comment]
gh pr checkout 324

File: README.md [hunk 1]
```diff
@@ -41,12 +41,12 @@ Pi.authenticate(scopes, onIncompletePaymentFound).then(function(auth) {
 });
 ```
 
-### Request a payment
```
Comment (blmitkc) [inline README.md:44]: []()@blmitkc ```
****
```

**Review by blmitkc — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (blmitkc) [inline README.md:44]:
```
@blmitkc []()@blmitkc 
```

**Review by blmitkc — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -67,14 +67,21 @@ Pi.createPayment({
 
 ```
 
-This code block is a **simplified example** to give you a sense of how it works.
```
Comment (blmitkc) [inline README.md:70]: Blagojmit@gmail.com **@wasi4440 ```

```**

**Review by blmitkc — CHANGES_REQUESTED** [review]

**Review by blmitkc — APPROVED** [review]

**Comment by seck6** [comment]
Snapchat-1752783396

**Comment by seck6** [comment]
Groupe

**Comment by ThePain813** [comment]
Look to be honest with u I don't understand this stuff I stumbled on this
somehow and I'm lost and confused.can you please write your messages in
English.i need help but I get scammed every time I ask someone to help
because I don't know this crypto like I should and I think I have a lot of
funds I unaware of

**Pull request closed without merging**